sample/Kmd.Logic.Gateway.Automation.Sample/Publish/publish.yml

```diff
@@ -61,15 +61,15 @@ Apis:
        - RevisionDescription: Rev2
          OpenApiSpecFile: ./apis/nexus2/v1/nexus-r2.json
          IsCurrent: true
-   Path: nexus2
- - Name: nexus3
+   Path: nexusxqk2
+ - Name: nexusxqk3
    ApiVersions:
     - VersionName: v1
       ApiLogoFile: ./apis/nexus3/nexus-v1.png
       ApiDocumentation: ./apis/nexus3/markdown-sample.md
       OpenApiSpecFile: ./apis/nexus3/nexus-v1.json
       Published: true
-      ProductNames: [nexus-2]
+      ProductNames: [nexusxqk-2]
       PolicyXmlFile: ./apis/nexus3/policies.xml
       BackendLocation: http://nexus3
-   Path: nexus3
\ No newline at end of file
+   Path: nexusxqk3
```

Comment:
Maybe let's not change those paths/names each time we want to test. We should clear our provider or do not push those changes.